server/lib/nl/common/bad_words.py

```diff
@@ -74,8 +74,8 @@ class BannedWords:
 # Loads a list of bad words from a text file.
 #
 def load_bad_words() -> BannedWords:
-  local_file = gcs.download_file(bucket=GLOBAL_CONFIG_BUCKET,
-                                 filename=BAD_WORDS_FILE)
+  local_file = gcs.maybe_download(
+      f'gs://{GLOBAL_CONFIG_BUCKET}/{BAD_WORDS_FILE}')
```

Comment:
Instead of hardcoding `gs://`, should we have a `make_path(bucket, suffix)` helper?